code_review.md

```diff
@@ -0,0 +1,34 @@
+Code Review Guidelines
+======================
+
+(some of the steps below don't necessarily need to be done in the order they are presented)
+
+1. Assign yourself to the pull request
+
+1. Verify that all of the GitHub checks have passed (Travis, etc.)
+
+1. Check out the branch in your local environment
+
+1. If you're reviewing code in an engine, regenerate the test app
+
+1. Verify that all the tests pass in your local environment
+
+1. Run the code in your local environment
+
+1. Verify that commits have been squashed and that the commit message properly describes the changes
+
+1. Read through the code changes and note any concerns
+
+1. Read through the tests.  Make sure that everything that needs a test has one.
```

Comment:
> Make sure that everything that needs a test has one.

This could probably use some guidelines - e.g.?:
* any public method of a new class should be unit tested
* logic in views should be tested
* expectations for testing JS
* Change should be accounted for in at least one feature test